social-tap/Form1.cs

```diff
@@ -45,6 +45,25 @@ private void SubmitClicked(object sender, EventArgs e)
             Console.WriteLine("Beverage level: " + beverageLevel/10);
             Console.WriteLine("Comment: " + comment);
             Console.WriteLine("User recommends: " + (recommends ? "true" : "false"));
+
+            Writter(barName, beverageLevel, comment, recommends);
+            
+        }
+
+       private void Writter(String barName, int beverageLevel, String comment, Boolean recommends)
```

Comment:
Nereikalinga funkcija, iš plain teksto skaitymas belenkaip sunkus bus. Užtenka kaip sakė kažkuris vedėjas pratybų konsolės, o po to iškart į DB, kas nesivargint su tais failais.